lesson05/assignment/css/style.scss

```diff
@@ -3,4 +3,94 @@
 @import 'settings.variables';
 @import 'settings.responsive';
 @import 'components.content';
-
+/* Use of Grids */
+main{
+  display:grid;
+  grid-template-rows:auto auto auto auto;
+}
+header{
+  background-color: rgb(241, 53, 43);
+  width:100%;
+  h1{
+  color: white;
+  font-family: HelveticaNeue, "Helvetica Neue";
```

Comment:
Font family list should be in descending order of specificity, so probably this should look something like: 
` font-family: "Helvetica Neue",  Helvetica, Arial, sans-serif;`